File: backend/src/main/java/es/codeurjc/backend/dto/ConcertDTO.java
package es.codeurjc.backend.dto;

import java.util.List;

public record ConcertDTO(
    Long id,
    String concertName,
    String concertDetails,
    String concertDate,
    String concertTime,
    String location,
    Integer stadiumPrice,
    Integer trackPrice,
    String map,
    boolean concertImage,
    String color,
    List<ArtistDTO> artistIds,
    List<TicketDTO> ticketIds
    
    ) {}

File: frontend/src/app/dtos/concert.dto.ts
import { ArtistDTO } from "./artist.dto";
import { TicketDTO } from "./ticket.dto";

export interface ConcertDTO {
	id?: number,
	concertName: string,
	concertDetails: string,
    concertDate: string,
    concertTime: string,
    location: string,
    stadiumPrice: number,
    trackPrice: number,
    map: string,
	concertImage: boolean,
    color: string,
    artists: ArtistDTO[],
    tickets: TicketDTO[]
}